docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -153,7 +153,8 @@ Instructions for creating your own service provider certificate can be found
 After your certificate is created you'll need to register it with the
 attestation service.
 `Click here <https://software.intel.com/formfill/sgx-onboarding>`_ for the
-registration form.
+registration form. Remeber to select ``Linkable`` when choosing the Attestation policy given that PoET relyes on "Linkable" EPID signatures. This means the device will always present a consistent identity within the Attestation service. Further details on Signature Policy can be found
```

Comment:
"relies" is misspelled as "relyes"

Reply:
"Remember" is misspelled as "Remeber".

Reply:
Suggestion: "Please select ``Linkable`` option when" would read better in lieu of "Remember to select ``Linkable`` when"

Reply:
"since" would be a better alternative for the "given that" in the text